### PR TITLE
feat: add thirdweb tutorials

### PR DESCRIPTION
**Pull request by avneesh0612**
# Tutorial submission

## Title of the tutorial
- Build a smart contract using thirdweb
- Build a dapp using thirdweb

## Brief tutorial description
The first guide includes creating and deploying a smart contract on zkSync and the second one includes creating and deploying a dapp.

## Checklist

- [x] I confirm that this tutorial is an original work and hasn't been published elsewhere.
- [x] I confirm that I am the rightful intellectual property owner (author) of this submission.
- [x] The tutorial includes referenced and relevant external sources (if applicable).
- [x] The tutorial includes working code snippets to illustrate concepts.
- [x] I have adhered to the tutorial guidelines provided in the repository.

### CODE REVIEW

**Comment by uF4No** [comment]
hi @avneesh0612 thanks for the PR!
I just noticed that you created addition `tutorials.json` files in each folder. The idea is to add all the tutorials in the `tutorials.json` file in the root of the project so we can list them in our docs from a single file. Can you update this?

File: tutorials/build-smart-contract-thirdweb/TUTORIAL.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+   - Providing the option to select which contract(s) you wish to deploy.
+   - Uploading your contract source code (ABI) to IPFS.
+
+2. When it is completed, it will open a dashboard interface to finish filling out the parameters.
+   - `_name`: contract name
+   - `_symbol`: symbol or "ticker"
+   - `_royaltyRecipient`: wallet address to receive royalties from secondary sales
+   - `_royaltyBps`: basis points (bps) that will be given to the royalty recipient for each secondary sale, e.g. 500 = 5%
+3. Select `ZksyncEra` as the network
+4. Manage additional settings on your contract’s dashboard as needed such as uploading NFTs, configuring permissions, and more.
```
Comment (idea404) [inline tutorials/build-smart-contract-thirdweb/TUTORIAL.md:89]: To me, after (creating an API key on the webUI and) running `npx thirdweb deploy` and filling the fields returned an error: 

```bash
Failed to deploy contract
Contract bytecode is invalid. 0x{"error":"Not Found"}
If you believe this is incorrect or the error persists, please reach out in [discord](https://discord.gg/thirdweb).
```

File: tutorials/build-smart-contract-thirdweb/tutorials.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[
```
Comment (idea404) [inline tutorials/build-smart-contract-thirdweb/tutorials.json:1]: same as with other JSON file

**Review by idea404 — CHANGES_REQUESTED** [review]
This PR needs a rewrite to be merged.

The smart contract tutorial is closer to a tutorial than the dapp tutorial. However, it does not follow through like a single storyline which I would vow for considering there could be new developers following this. Moreover, the deployment failed for me with an error message which I hope is an anomaly as it could also throw other readers off. 

The dApp tutorial is far from a tutorial at the moment, it mentions several functionalities without linking them together in a development flow which is more akin to a ref doc than a tutorial. I strongly suggest changing this to a story format such that readers may follow along to experience the tooling.

File: tutorials/build-smart-contract-thirdweb/TUTORIAL.md (hunk 1 above)
Comment (uF4No) [reply tutorials/build-smart-contract-thirdweb/TUTORIAL.md:89]: @avneesh0612 can you confirm if `npx thidweb deploy` uses the zksolc compiler behind the scenes? All contracts deployed on zkSync Era must be compiled with our compilers

**Comment by idea404** [comment]
Closes #11

**Pull request closed without merging**